xcodecommand/xcpretty_helper.go

```diff
@@ -0,0 +1,39 @@
+package xcodecommand
+
+import (
+	"github.com/bitrise-io/go-steputils/v2/ruby"
+	command "github.com/bitrise-io/go-utils/v2/command"
+	version "github.com/hashicorp/go-version"
+)
+
+type xcprettyManager interface {
```

Comment:
Wouldn't it make sense to merge this whole helper file into `xcpretty_installer.go`? These methods mostly do delegation to other methods.

Reply:
Yes, moved it. Only split in a new file so that the diff engine picks it as a moved file but it did not.